notebooks/nvidia_haystack_medical_chatbot.ipynb

```diff
@@ -0,0 +1,322 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #27.    keyword_llm.warm_up()
</pre><p>JFYI, you don't need to call <code>warm_up</code> if you're going to use these generators in a pipeline as Pipeline calls it when it runs but no harm keeping these calls   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>